packages/data-explorer-ui/src/components/Filter/components/FilterNoResultsFound/filterNoResultsFound.tsx

```diff
@@ -20,15 +20,19 @@ export const FilterNoResultsFound = ({
       <Typography
         component="div"
         color="ink.light"
-        mb={2}
+        mb={onClearSearchTerm ? 2 : 0}
         mt={1}
         variant="text-body-400"
       >
         Try adjusting your search or filter to find what you’re looking for.
       </Typography>
-      <TextButtonPrimary onClick={onClearSearchTerm}>
-        Clear All
-      </TextButtonPrimary>
+      {onClearSearchTerm ? (
+        <TextButtonPrimary onClick={onClearSearchTerm}>
+          Clear All
+        </TextButtonPrimary>
+      ) : (
+        ""
```

Comment:
If `onClearSearchTerm` is not defined, we can return `null`, so an alternative to https://github.com/clevercanary/data-explorer/pull/280/files#r1267366143 would be:

```
{onClearSearchTerm ? (<TextButtonPrimary onClick={onClearSearchTerm}>Clear All</TextButtonPrimary>) : null}
```